README.md

```diff
@@ -58,37 +63,32 @@ windows.
 
 ### Subscribe to events
 
-Subscribing to events is also simple. The following example tells Chasqui to
-forward events to this subscriber's `my-app` queue. Chasqui will detect whether
-you are using Sidekiq or Resque and generate an appropriate worker class for you.
-Within the subscriber block, you use the `on(event_name)` method to register blocks
-of code to handle that event.
+Subscribing to events is also simple. In the following example, we create a
+subscriber to handle events published to the `user.sign-up` channel.
 
     # file: subscriber1.rb
     require 'chasqui'
 
-    Chasqui.subscribe queue: 'my-app' do
+    class UserSignUpSubscriber
+      include Chasqui::Subscriber
+      subscribe channel: 'user.sign-up'
```

Comment:
@kellyredding That's one of the things that I had a hard time picking up when I first started trying to use Chasqui. But yeah, really the vocabulary is just that, vocabulary. It can be ironed out once you're happy with the system.